TESTING.md

```diff
@@ -552,11 +552,17 @@ Apart from using Gradle, it is also possible to gain insight in code coverage us
 
 Please read your IDE documentation for how to attach a debugger to a JVM process.
 
-# Building with extra plugins
+# Testing with plugins
```

Comment:
Update TOC at the top of this document too, please.

Reply:
Missed that, thanks!